ligo/haskell/test/Test/Ligo/BaseDAO/Management.hs

```diff
@@ -163,3 +164,268 @@ test_BaseDAO_Management =
+        (#newOwner .! wallet1)
+      withSender (AddressResolved wallet2) $ call baseDao (Call @"Accept_ownership") ()
+        & expectNotPendingOwner
+
+changeToPendingAdmin
+  :: forall caps base m param pm st
+  . (MonadNettest caps base m, ParameterC param pm)
+  => WithOriginateFn m param st -> WithStorage st -> m ()
+changeToPendingAdmin withOriginatedFn initialStorage =
+  withOriginatedFn 2 (\(owner:_) -> initialStorage owner) $
+    \[owner, wallet1] baseDao -> do
+      withSender (AddressResolved owner) $ call baseDao (Call @"Transfer_ownership")
+        (#newOwner .! wallet1)
+      withSender (AddressResolved wallet1) $ call baseDao (Call @"Accept_ownership") ()
+
+noPendingAdmin
+  :: forall caps base m param pm st
+  . (MonadNettest caps base m, ParameterC param pm)
+  => WithOriginateFn m param st -> WithStorage st -> m ()
+noPendingAdmin withOriginatedFn initialStorage =
+  withOriginatedFn 2 (\(owner:_) -> initialStorage owner) $
+    \[_, wallet1] baseDao -> do
+      withSender (AddressResolved wallet1) $
+        call baseDao (Call @"Accept_ownership") ()
+        & expectNotPendingOwner
+
+pendingOwnerNotTheSame
+  :: forall caps base m param pm st
+  . (MonadNettest caps base m, ParameterC param pm)
+  => WithOriginateFn m param st -> WithStorage st -> m ()
+pendingOwnerNotTheSame withOriginatedFn initialStorage =
+  withOriginatedFn 2 (\(owner:_) -> initialStorage owner) $
+    \[owner, wallet1] baseDao -> withSender (AddressResolved owner) $ do
+      call baseDao (Call @"Transfer_ownership")
+        (#newOwner .! wallet1)
+      call baseDao (Call @"Accept_ownership") ()
+      & expectNotPendingOwner
+
+acceptOwnerRespectMigration
+  :: forall caps base m param pm st
+  . (MonadNettest caps base m, ParameterC param pm)
+  => WithOriginateFn m param st -> WithStorage st -> m ()
+acceptOwnerRespectMigration withOriginatedFn initialStorage =
+  withOriginatedFn 2 (\(owner:_) -> initialStorage owner) $
+    \[owner, newAddress1] baseDao -> do
+      withSender (AddressResolved owner) $
+        call baseDao (Call @"Migrate") (#newAddress .! newAddress1)
+      -- We test this by calling `confirmMigration` and seeing that it does not fail
+      withSender (AddressResolved newAddress1) $
+        call baseDao (Call @"Confirm_migration") ()
+      withSender (AddressResolved owner) $ call
+        baseDao (Call @"Accept_ownership") ()
+        & expectMigrated newAddress1
+
+
+migrationAuthenticateSender
+  :: forall caps base m param pm st
+  . (MonadNettest caps base m, ParameterC param pm)
+  => WithOriginateFn m param st -> WithStorage st -> m ()
+migrationAuthenticateSender withOriginatedFn initialStorage =
+  withOriginatedFn 3 (\(owner:_) -> initialStorage owner) $
+    \[_, newAddress1, randomAddress] baseDao -> do
+      withSender (AddressResolved randomAddress) $
+        call baseDao (Call @"Migrate") (#newAddress .! newAddress1)
+        & expectNotAdmin
+
+migrationSetPendingOwner
```

Comment:
How does this test prove that migration sets pending owner?

Reply:
Good catch, I think this was just poorly named, changed it to `migrationSetTarget`